src/Arcus.Observability.Telemetry.Core/MessageFormats.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Arcus.Observability.Telemetry.Core
+{
+    /// <summary>
+    /// Represents the publicly available message formats to track telemetry.
+    /// </summary>
+    public static class MessageFormats
```

Comment:
Maybe consider moving all formats?

Reply:
Yeah, maybe that's a better approach. More consistent too.

Reply:
Will still use the 'old' message format for now to track events, will change it in #167 .

Reply:
Done this, resolving.